boards/README.md

```diff
@@ -21,3 +21,14 @@ that Tock supports.
 | [SiFive HiFive1](hifive1/README.md)                                  | RISC-V          | FE310-G000     | openocd    | tockloader     | [Yes (5.1)][qemu] |
 | [Digilent Arty A-7 100T](arty_e21/README.md)                         | RISC-V RV32IMAC | SiFive E21     | openocd    | tockloader     | No                |
 | [Nexys Video OpenTitan](opentitan/README.md)                         | RISC-V RV32IMC  | EarlGrey       | custom     | custom         | [Yes (5.1)][qemu] |
+
+# Out of Tree Boards
+
+Some projects that use Tock maintain their own board definitions outside the
+Tock repository.
+
+| Project                                                  | Boards                       | Architecture   | MCU      | Build System  |
+|----------------------------------------------------------|------------------------------|----------------|----------|---------------|
+| [OpenSK](https://github.com/google/opensk)               | nRF52840-DK, nRF52840-Dongle | ARM Cortex-M4  | nRF52840 | Python script |
```

Comment:
OpenSK also adds support for the nrf52840-mdk (https://makerdiary.com/products/nrf52840-mdk-iot-development-kit) - see https://github.com/google/OpenSK/tree/master/boards/nordic/nrf52840_mdk_dfu

Reply:
Good catch! Added.